admin/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^4.1.0",
+    "jquery": "^3.3.1",
+    "popper.js": "^1.14.3",
+    "vue": "^2.1.10"
+  }
```

Comment:
You use Vue.JS??!

Reply:
lol, I was going to but I didn't have time learning vue.